### ISSUE TITLE
Action Potential templates not centered at the peak

### ISSUE THREAD

**Issue by srarty** [cedavey/SpikeExtractionToolGit#8]
The action potentials are centred according to some zero-cross criteria. Sometimes there is some offset in the noise, causing the template to be centred at a random position, hence some APs from the same template are out of phase. We might need to center at the peak instead.

**Comment by cedavey** [comment]
The APs have always been centred at the peak, so I'm unsure what change is required.

**Comment by srarty** [comment]
That's what I thought as well. I will assign the issue to myself and double check what is the root of the problem

**Comment by cedavey** [comment]
Do you mean that the problem is with the spikes within a template? I'm currently changing the templates to allow for different length spikes so we can wait to see if it gets fixed just by me changing how the templates managed. If it's spikes rather than spikes within a template that is the problem then please go ahead.

**Comment by srarty** [comment]
Yes. And I think allowing different lengths might do the trick, probably.

The issue is with the spikes within a template, as in, the stored data. The problem is not with displaying the APs on the AP view, but with the register of each spike in a template.
